README.md

```diff
@@ -103,22 +107,13 @@ Connecting an SMT with an RMT this way is not supported.
 
 - `rmt-cli import repos /mnt/usb` will mirror all repos which are enabled in the database, from the given path.
 
-
-### openSUSE and other RPM based products
-
-To mirror repositories that are not delivered via SCC, you can run for example:
-
-`rmt-cli mirror custom https://download.opensuse.org/repositories/systemsmanagement:/SCC:/RMT/openSUSE_Leap_42.3/ foo/bar`
```

Comment:
The only way to do that now is setting up the custom repositories via `rmt-cli repos custom` commands.